web-app/js/portal/filter/Filter.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2015 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+Ext.namespace('Portal.filter');
+
+Portal.filter.Filter = Ext.extend(Object, {
+
+    constructor: function(filterDetail, layer) {
+
+        this.layer = layer;
+        this.filterName = filterDetail.name;
```

Comment:
We've actually been a little inconsistent in both JS and Groovy land. We should tidy this up as we do this work.